src/sentry/integrations/jira/integration.py

```diff
@@ -560,6 +563,9 @@ def create_issue(self, data, **kwargs):
             'key': issue_key,
         }
 
+    def merge_issue_config_defaults(self, project_id, config):
+        return config
```

Comment:
@macqueen this change should also make sure dynamic fields still work as intended